### PR TITLE
tvOS support

### PR DESCRIPTION
**Pull request by zachmanc**
- added support for tvOS
 - upgraded to ComScore version 5.8.4 that supports tvOS

### CODE REVIEW

File: BitmovinComScoreAnalytics.podspec [hunk 1]
```diff
@@ -6,6 +6,7 @@ Pod::Spec.new do |s|
   Comscore Analytics Integration for the Bitmovin Player iOS SDK
                        DESC
 
+  s.swift_version    = '4.0'
```
Comment (stonko1994) [inline BitmovinComScoreAnalytics.podspec]: Why not using latest swift version `4.2`?

File: BitmovinComscoreAnalytics/Classes/ComScoreBitmovinAdapter.swift [hunk 2]
```diff
@@ -80,10 +80,14 @@ extension ComScoreBitmovinAdapter: PlayerListener {
     }
 
     func onPaused(_ event: PausedEvent) {
+        
+        //TODO: remove once we have tvOS support for this method
+        #if(iOS)
```
Comment (stonko1994) [inline BitmovinComscoreAnalytics/Classes/ComScoreBitmovinAdapter.swift]: Does this even work? I assume this macro check is not working.

It should be `#if os(iOS)`

File: BitmovinComscoreAnalytics/Classes/ComScoreBitmovinAdapter.swift [hunk 3]
```diff
@@ -114,11 +118,14 @@ extension ComScoreBitmovinAdapter: PlayerListener {
     }
     
     private func resume(){
+        //TODO remove once we have iOS support
+        #if(iOS)
```
Comment (stonko1994) [inline BitmovinComscoreAnalytics/Classes/ComScoreBitmovinAdapter.swift]: same as above

File: BitmovinComscoreAnalytics/Classes/ComScoreBitmovinAdapter.swift [hunk 4]
```diff
@@ -114,11 +118,14 @@ extension ComScoreBitmovinAdapter: PlayerListener {
...
         if player.isAd {
             playAdContentPart(duration: currentAdDuration, timeOffset: currentAdOffset)
         }else {
```
Comment (stonko1994) [inline BitmovinComscoreAnalytics/Classes/ComScoreBitmovinAdapter.swift]: Please fix `swiftLint` complains

File: Example/BitmovinComScoreAnalytics_Example_tvOS/AppDelegate.swift [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+    var window: UIWindow?
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        return true
+    }
+
+    func applicationWillResignActive(_ application: UIApplication) {
```
Comment (stonko1994) [inline Example/BitmovinComScoreAnalytics_Example_tvOS/AppDelegate.swift]: Remove unnecessary methods

File: Example/BitmovinComscoreAnalytics/Info.plist [hunk 6]
```diff
@@ -3,7 +3,7 @@
 <plist version="1.0">
 <dict>
 	<key>BitmovinPlayerLicenseKey</key>
-	<string></string>
+	<string>18ca6ad5-9768-4129-bdf6-17685e0d14d2</string>
```
Comment (stonko1994) [inline Example/BitmovinComscoreAnalytics/Info.plist]: Please remove license key. (Also change commit)

File: BitmovinComScoreAnalytics.podspec (hunk 1 above)
Comment (zachmanc) [reply BitmovinComScoreAnalytics.podspec]: confirmed Turner was okay moving to 4.2

**Comment by zachmanc** [comment]
@stonko1994 please take another look, thanks!

File: Example/.swiftlint.yml [hunk 7]
```diff
@@ -14,7 +14,7 @@ excluded:
 - Pods
 
 line_length:
-    warning: 150
```
Comment (stonko1994) [inline Example/.swiftlint.yml]: Usually we have 120. I know that this is hard for urls but therefore we disable it for the line explicitly.

_I would prefer to change to 120 (or 150). But 200 is to high in my opinion_

File: Example/BitmovinComScoreAnalytics_Example_tvOS/ViewController.swift [hunk 8]
```diff
@@ -8,14 +8,77 @@
...
+        do {
+            try config.setSourceItem(url: streamUrl)
+            
+            // Create player based on player configuration
+            let player = BitmovinPlayer(configuration: config)
+            self.bitmovinPlayer = player
+            
+            self.bitmovinPlayer?.add(listener: self)
+            // Create a Comscore Configuration
+            let comScoreMetadata: ComScoreMetadata = ComScoreMetadata(mediaType: .longFormOnDemand, publisherBrandName: "ABC", programTitle: "Modern Family", episodeTitle: "Rash Decisions", episodeSeasonNumber: "1", episodeNumber: "2", contentGenre: "Comedy", stationTitle: "Hulu", completeEpisode: true)
```
Comment (stonko1994) [inline Example/BitmovinComScoreAnalytics_Example_tvOS/ViewController.swift]: Change to multiline. Each property in a new line (starting with `publisherBrandName`) aligned with the first one (`mediaType`).

File: Example/BitmovinComscoreAnalytics/Info.plist [hunk 9]
```diff
@@ -3,7 +3,7 @@
 <plist version="1.0">
 <dict>
 	<key>BitmovinPlayerLicenseKey</key>
-	<string>18ca6ad5-9768-4129-bdf6-17685e0d14d2</string>
```
Comment (stonko1994) [inline Example/BitmovinComscoreAnalytics/Info.plist]: Shouldn't we change git history to remove the key completely?

File: Example/.swiftlint.yml (hunk 7 above)
Comment (zachmanc) [reply Example/.swiftlint.yml]: changed to 120

**Pull request merged**